README.md

```diff
@@ -6,43 +6,43 @@
 This is a library implementing a simple pattern that encourages modularity and
 the Single Responsibility Principle around _doing_ things primarially with ecto.
 
-The goal is a module whose sole responsibility is doing one thing, removing that
-responsibility from the schema (model) module. Some examples might be:
-
-* CreatePost
-* UpdatePost
-* DeletePost
+You can think of this as a second layer of domain modeling that happens in your
+application. You existing schema (model) layer represents the data itself and
+it's relationships. The new interaction layer represents high level actions or
+events that happen on your domain. Some good examples in a blog domain might be:
```

Comment:
Thanks, I agree that complicated interactions are where a tool like this is useful. The events I listed are intended to be simple names for a whole bunch of related functionality. In fact I would go so far as to say the point of the library is to wrap complex tasks into simple concepts.  (For example: CreateArticle could format content, create a record, update a metric, update counter columns, queue for moderation, and publish to a websocket).

I probably won't update the docs now (so many things to do), but I would love a PRs that improves the docs!